### PR TITLE
boilerplate

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

File: k8s/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* copy output of `config.json` secret into `k8s-config.yaml`
+
+## develop
+
+* run `skaffold dev` in the root of the project
+
+* update your `hosts` file to point spindev.internal.yale.yale.edu to localhost
+
+* use the endpoint `http://<<spindev.internal.yale.edu>>/v1/ds`
```
Comment (tenyo) [inline k8s/README.md]: 👀 `ds`

File: api/routes.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+func (s *server) routes() {
+	api := s.router.PathPrefix("/v1/ds").Subrouter()
```
Comment (tenyo) [inline api/routes.go]: 👀 `ds`

File: api/server.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+	if config.Org == "" {
+		return errors.New("'org' cannot be empty in the configuration")
+	}
+	Org = config.Org
+
+	publicURLs := map[string]string{
+		"/v1/ds/ping":    "public",
+		"/v1/ds/version": "public",
+		"/v1/ds/metrics": "public",
```
Comment (tenyo) [inline api/server.go]: 👀 😮

**Comment by fishnix** [comment]
I fixed those on my current branch, so I'm going to merge this in and I'll PR those with the next PR.

**Pull request merged**